pkgs/applications/misc/orca/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv, lib, pkgconfig, fetchFromGitHub, pkgs
+, autoreconfHook, wrapGAppsHook
+, intltool, yelp-tools, itstool, libxmlxx3
+, python, pygobject3, gtk3
+, at_spi2_atk, at_spi2_core, pyatspi, dbus, dbus-python, pyxdg
+, louis ? null
+, xmodmap, xkbcomp, gsettings_desktop_schemas
+
+, speechdSupport ? true, speechd
+, brailleSupport ? false, brltty, brlapi ? null
+, setproctitleSupport ? true, setproctitle
+, gstreamerSupport ? true, gst_all_1, gst-python
+}:
+
+with lib;
+#pkgs.python3Packages.buildPythonPackage rec {
```

Comment:
- Import directly `python3Packages` instead of using `pkgs`.
- `buildPythonApplication` should be used instead of `buildPythonPackage`.
- You need to add `format = "other"` to be able to build it.

Reply:
I took it one step further and imported `buildPythonApplication`.
`format = "other";` did solve the error about not finding `setyp.py` indeed.